install_config/install/quick_install.adoc

```diff
@@ -19,87 +19,274 @@ toc::[]
 
 == Overview
 The _quick installation_ method allows you to use an interactive CLI utility to
-install OpenShift across a set of hosts. The installation utility is provided to
-make the trial experience easier by interactively gathering the data needed to
-run the installation. The utility is a self-contained wrapper available at
-https://install.openshift.com and is intended for usage on a Red Hat Enterprise
-Linux 7 host.
+install OpenShift across a set of hosts. The installation utility can deploy
+OpenShift components on targeted hosts by installing RPMs.
+
+This installation method is provided to make the installation experience easier
+by link:#running-an-interactive-installation[interactively gathering the data]
+needed to run on each host. The utility is a self-contained wrapper intended for
+usage on a Red Hat Enterprise Linux (RHEL) 7 system. While RHEL Atomic Host is
+supported for running containerized OpenShift services, the installation utility
+is
+link:../../install_config/install/prerequisites.html#software-prerequisites[provided
+by an RPM] not available by default in RHEL Atomic Host, and must therefore be
+run from a RHEL 7 system. The host initiating the installation does not need to
```

Comment:
Containerized installations were introduced as Tech Preview in OSE 3.1, and became fully supported in OSE 3.1.1. So you can take out this "While RHEL Atomic Host is supported..." sentence, and any reference to containerized installs as well (looks like you already got any others).